components/form.js

```diff
@@ -370,6 +373,7 @@ export function MarkdownInput ({ label, topLevel, groupClassName, onChange, onKe
                 const s3Keys = [...text.matchAll(AWS_S3_URL_REGEXP)].map(m => Number(m[1]))
                 updateUploadFees({ variables: { s3Keys } })
                 setSubmitDisabled?.(false)
+                window.localStorage.setItem(storageKey, text)
```

Comment:
Since apparently `storageKey` can be undefined, you should also check for that case here (like is done in `onChangeInner`).

Reply:
Sorry, you are right, I totally missed it.